main.go

```diff
@@ -181,6 +178,7 @@ See https://kairos.io/docs/upgrade/manual/ for documentation.
 			}
 
 			return checkRoot()
+			return nil
```

Comment:
?

Reply:
damn, local testing to see if the command did pass the args lmao